### PR TITLE
Remove unused ignore rule

### PR DESCRIPTION
**Pull request by Frederisk**
In the previous PR, this file is no longer used.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,3 @@
-config.env
```
Comment (leemgs) [inline .gitignore:1]: I think that this line have to be replaced "config.env" with ".env" instead of the removal.

Comment (Frederisk) [reply .gitignore:1]: What? `.env` is already in the content after.

Comment (Frederisk) [reply .gitignore:1]: What? `.env` is already in the content after.

Comment (moyix) [reply .gitignore:1]: Yep, it's down near the bottom of the file.

**Comment by moyix** [comment]
LGTM, thanks!

**Pull request merged**